File: Python_Solutions/CodeWars/8kyu/Beginner_Reduce_but_Grow.py
# Given a non-empty array of integers, return the result of multiplying the values together in order. Example:
#
# [1, 2, 3, 4] => 1 * 2 * 3 * 4 = 24
# FUNDAMENTALSARRAYS
# Solution
def grow(arr):
    integer = 1
    for elem in arr:
        integer *= elem
    return integer

File: CodeWars/8kyu/Beginner_Reduce_but_Grow.cpp
/*
Given a non-empty array of integers, return the result of multiplying the values together in order. Example:

[1, 2, 3, 4] => 1 * 2 * 3 * 4 = 24
FUNDAMENTALSARRAYS
*/
// Solution
#include <vector>
int grow(std::vector<int> nums) {
  int accumulate = 1;
  for (int num : nums) {
    accumulate *= num;
  }
  return accumulate;
}